packages/zcli-themes/src/lib/pollJobStatus.ts

```diff
@@ -5,7 +5,7 @@ import { request } from '@zendesk/zcli-core'
 import * as chalk from 'chalk'
 import validationErrorsToString from './validationErrorsToString'
 
-export default async function pollJobStatus (themePath: string, jobId: string, interval = 1000, retries = 10): Promise<void> {
+export default async function pollJobStatus (themePath: string, jobId: string, interval = 2000, retries = 10): Promise<void> {
```

Comment:
Minot nit: could we instead increase the number of retries?
Reason for it being that we'd still ping more often but for longer so we get the new status faster.

Reply:
I think at the time we mentioned both options. I believe we can increase the retries instead if pinging more times is ok. What do you think about increasing it to `15`?